src/mongocxx/test/collection.cpp

```diff
@@ -532,6 +532,69 @@ TEST_CASE("CRUD functionality", "[driver::collection]") {
         auto results = coll.aggregate(p);
     }
 
+    SECTION("aggregation $lookup operator", "[collection]") {
+        auto people_coll_name = "people_on_the_block";
+        auto people_coll = db.create_collection(people_coll_name);
+        auto houses_coll_name = "houses_on_the_block";
+        auto houses_coll = db.create_collection(houses_coll_name);
+
+        // populate one collection with names
+        document name1;
```

Comment:
You don't have to rewrite it, but more stylish:

auto name1 = document{} << "key" << "value" << finalize;